doc/040_backup.rst

```diff
@@ -355,6 +355,7 @@ environment variables. The following list of environment variables:
     RESTIC_REPOSITORY                   Location of repository (replaces -r)
     RESTIC_PASSWORD_FILE                Location of password file (replaces --password-file)
     RESTIC_PASSWORD                     The actual password for the repository
+    RESTIC_PASSWORD_COMMAND             Command printing the password for the repository to stdout
```

Comment:
Please:

- Move this line up to right above the RESTIC_PASSWORD_FILE line.
- Change the text to something like "Command printing the repository password to stdout (replaces --password-command)"

Thank you!

Reply:
> Change the text to something like "Command printing the repository password to stdout (replaces --password-command)"

That's not quite right, if the environment variable is present, its value is used as the default value for `--password-command`. If `--password-command` is specified, that value is used instead of the environment variable. :)

Reply:
Thanks for correcting me. Then there's no point in spending time on the other cosmetic things, there'll be a merge commit anyway.